accounts-db/src/accounts_index/in_mem_accounts_index.rs

```diff
@@ -1482,35 +1307,8 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> InMemAccountsIndex<T,
             return;
         }
 
-        let stop_evictions_changes_at_start = self.get_stop_evictions_changes();
         let next_age_on_failure = self.storage.future_age_to_flush(false);
-        if self.get_stop_evictions() {
-            // ranges were changed
-            self.move_ages_to_future(next_age_on_failure, current_age, &evictions);
-            return;
-        }
-
         let mut failed = 0;
-
-        // skip any keys that are held in memory because of ranges being held
```

Comment:
There is no longer `cache_range_held` after we remove hold range in memory concept. 
So we can get rid of this code.